src/tests.builds

```diff
@@ -22,7 +22,7 @@
     <SerializeProjects>true</SerializeProjects>
   </PropertyGroup>
 
-  <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
+  <Import Project="$(ToolsDir)tests.targets" Condition="Exists('$(ToolsDir)tests.targets')" />
```

Comment:
Importing this might cause bad things to happen because of incorrect assumptions. We need to be careful and be sure doing so doesn't end up causing this to be like a test project or invalid the normal traversal Test target from working. 

If you do continue to import this then it already imports PerfTest.targets